infolettre/infolettre_16/index.qmd

```diff
@@ -0,0 +1,436 @@
+très différentes: Sam Altman, le PDG, 
+était licencié par le conseil d'administration
+avant d'être réembauché. 
+Parmi les hypothèses pour expliquer cette crise,
+l'une des plus récurrentes serait des
+désaccords entre le comité d'administration et les équipes
+d'OpenAI sur les recherches menées dans le cadre d'un projet
+nommé $Q^*$ d'une intelligence artificielle en mesure de résoudre
+des problèmes de logiques mathématiques, un sujet sur lequel 
+les meilleures IA actuelles sont défaillantes. 
+
+::: {.callout-note}
+## Pour en savoir plus sur $Q^*$:
+
+- Une [dépêche de _Reuters_](https://www.reuters.com/technology/sam-altmans-ouster-openai-was-precipitated-by-letter-board-about-ai-breakthrough-2023-11-22/)
+
+:::
+
+## L'Europe adopte un _"AI Act"_ historique
+
+Les Etats membres et le Parlement européen sont parvenus à un accord le 9 décembre
+sur des règles régissant les systèmes d’intelligence artificielle dans un accord nommé
+"Artificial Intelligence Act". 
+Cet accord est un compromis entre une régulation forte
+protégeant les données produites quotidiennement
+par les citoyens européens
+et une approche plus permissive, défendue
+par exemple par la France et l'Allemagne,
+pouvant favoriser l'innovation dans le domaine.
+
+Débutées en 2018, avant que les IA génératives ne deviennent
+si populaires, ces discussions dépassent le cadre exclusif de ces
+dernières. Néanmoins,
+concernant celles-ci, le compromis prévoit une approche à deux vitesses. 
+Outre le respect des règles européennes 
+de propriété intellectuelle, les développeurs de modèles génératifs
+devront s'assurer que les produits diffusés soient bien identifiés comme
+artificiels, afin de limiter la
+diffusion de [_deepfakes_](https://fr.wikipedia.org/wiki/Deepfake).
+Les développeurs de ces modèles
+devront également communiquer sur la qualité des données utilisées
+pour entraîner les modèles et sur le coût énergétique de ceux-ci. 
+Les modèles _open source_ et ceux construits à des fins de recherche bénéficient
+d'exemptions de ces règles. 
+
+Des contraintes renforcées s’appliqueront aux systèmes jugés à _"haut risque"_
+dans des domaines comme la défense, l'éducation, les ressources humaines
+ou encore la santé. 
+Pour ces systèmes, il sera nécessaire de réaliser une analyse d’impact avant la mise sur le marché. Par ailleurs, une obligation de transparence et d’explicabilité des modèles est mise en place
+afin d'être en mesure de comprendre les règles de décision de ces IA.
+
+L'AI Act interdit également l'utilisation de l'IA dans quelques domaines, jugés trop sensibles. 
+Par exemple, la reconnaissance faciale de masse est interdite, hormis lorsque celle-ci
+est justifiée par des motifs de sécurité nationale.  
+D'autres utilisations, porteuses potentiellement de dérives, comme
+la notation sociale basée sur le comportement ou des caractéristiques personnelles
+sont interdits. 
+
+::: {.callout-note}
+## Pour en savoir plus
```

Comment:
Ce sujet est vraiment sensible. Andrew Ng a même fait une déclaration devant le Sénat américain sur ce thème. Je pense qu'ici on doit se contenter d'informer sur le contenu du document et les enjeux associés, comme tu l'as fait plus haut. Cela pourrait être utile toutefois d'ajouter la référence directe à la page associée sur le site de la commission. A ce stade il bugue malheureusement... https://www.europarl.europa.eu/news/fr/headlines/society/20230601STO93804/loi-sur-l-ia-de-l-ue-premiere-reglementation-de-l-intelligence-artificielle

Reply:
on peut mettre ce lien à la place

https://www.consilium.europa.eu/fr/press/press-releases/2023/12/09/artificial-intelligence-act-council-and-parliament-strike-a-deal-on-the-first-worldwide-rules-for-ai/

Reply:
en revanche, j'ai pas compris s'il fallait réécrire le paragraphe ou pas.